src/common/hotbar-store.ts

```diff
@@ -153,6 +153,13 @@ export class HotbarStore extends BaseStore<HotbarStoreModel> {
     }
   }
 
+  @action
+  setHotbarName(id: string, name: string) {
+    const idx = this.hotbars.findIndex((hotbar) => hotbar.id === id);
```

Comment:
This should check for unknown id

Reply:
Oki, should I throw an error or just `return` undefined?

Reply:
I would log a warning and then return undefined

Reply:
Okay, not sure we have to check for unknown id, because before calling `setTorbarName` we check hotbar existence  here.. 
https://github.com/lensapp/lens/pull/3469/files/62e42c9774d8e2c16449e172119c98fb914c1ac3#diff-9c1d174748a991f0ca88f6194e58e846475d55bf4228295f3a741482582e870fR59

Reply:
It is just better practice. We may add calls in the future that don't do that check, or they might be some other weirdness.